apps/spruce/src/analytics/waterfall/useWaterfallAnalytics.ts

```diff
@@ -38,7 +38,8 @@ type Action =
   | {
       name: "Viewed waterfall modal";
       navigated_to_waterfall: boolean;
-    };
+    }
+  | { name: "System Event redirect"; referrer: string };
```

Comment:
I'm not sure what system event means but to be more in line with our other events I would reword it slightly
```suggestion
  | { name: "Redirected via system event"; referrer: string };
```

Reply:
I forgot that redirected was one of our analytics event actions thank you!